src/main/java/in/oogway/ologger/oogwayLogger.java

```diff
@@ -0,0 +1,8 @@
+package in.oogway.ologger;
+
+/*** Created by swapnas on 7/24/17.*/
+public interface oogwayLogger {
+    void setLevel(Level level);
+    Level getLevel();
+    void log(String msg);
```

Comment:
OogwayLogger should also expose log debug warn critical etc. Since it's common to both implementations.